Formula/algernon.rb

```diff
@@ -1,12 +1,24 @@
 class Algernon < Formula
   desc "Pure Go web server with Lua, Markdown, HTTP/2 and template support"
   homepage "https://github.com/xyproto/algernon"
-  url "https://github.com/xyproto/algernon/archive/refs/tags/v1.15.2.tar.gz"
-  sha256 "eb693954feaac8e589818f41c619c362256bac774e57c69d24c4ab08201974ee"
   license "BSD-3-Clause"
+  revision 1
   version_scheme 1
   head "https://github.com/xyproto/algernon.git", branch: "main"
 
+  stable do
+    url "https://github.com/xyproto/algernon/archive/refs/tags/v1.15.2.tar.gz"
+    sha256 "eb693954feaac8e589818f41c619c362256bac774e57c69d24c4ab08201974ee"
+
+    # Add support for Go 1.20 and 1.21 from
+    # https://github.com/xyproto/algernon/pull/136
+    # can likely be removed for algernon >1.15.2
+    patch do
+      url "https://github.com/xyproto/algernon/commit/cd3d98354745ee6dd3fd17a3327269dcf19bf434.diff?full_index=1"
```

Comment:
This should be `.patch`:
```suggestion
      url "https://github.com/xyproto/algernon/commit/cd3d98354745ee6dd3fd17a3327269dcf19bf434.patch?full_index=1"
```
...but server returns 403 with message:
````
Content containing PDF or PS header bytes cannot be rendered from this domain for security reasons.
````
What to do in such case?

Reply:
also, that seems like a bit patch, we can probably just wait for their release other than applying this patch